src/services/log.service.js

```diff
@@ -30,30 +30,12 @@ let logService = () => {
     ])
   }
 
-  let getByStatus = (status, page, pageSize) => {
```

Comment:
As for today, it's still unclear what we're storing as status. Once we define the purpose of this field, we can add it back.